public/app/features/alerting/AlertTabCtrl.ts

```diff
@@ -232,6 +240,23 @@ export class AlertTabCtrl {
     this.panelCtrl.render();
   }
 
+  checkFrequency() {
+    this.frequencyWarning = '';
+
+    try {
+      const frequencySecs = kbn.interval_to_seconds(this.alert.frequency);
+      if (frequencySecs < this.alertingMinIntervalSecs) {
+        this.frequencyWarning =
+          'The rule will be evaluated every ' +
```

Comment:
Would like to change the warning text, but not exactly sure to what. Suggestions:
- "A minimum evaluation interval of `this.alertingMinInterval` have been configured and will be used for this rule"
- "A minimum evaluation interval have been configured. This rule will be evaluated every `this.alertingMinInterval`"

@oddlittlebird maybe have some input on the warning text?

Evaluation interval is what we're referring to in docs: https://grafana.com/docs/grafana/latest/alerting/rules/#name-and-evaluation-interval

Reply:
Both of those warning texts are very passive. It makes it hard to understand what is happening to what. Of the two listed, the second is better. I would suggest something like:

"<rule name> will be evaluated every <this.alertingMinInterval>. You can change this by adjusting the <whatever> value."

Either that or the second sentence should be other useful information about why they might want to increase or lower the value.